app/components/elements/forms/repeatable_nested_component.html.erb

```diff
@@ -2,11 +2,11 @@
   <legend class="h5">
     <%= render Elements::Forms::LabelComponent.new(form:, field_name:, label_text:, hidden_label:) %>
   </legend>
-  <%= tag.div(class: 'mb-3 container', data: { nested_form_target: 'container' }) do %>
+  <%= tag.div(class: container_classes, data: { nested_form_target: 'container' }) do %>
     <template data-nested-form-target="template">
       <%= form.fields_for field_name, model_class.new, child_index: 'NEW_RECORD' do |nested_form| %>
         <%# Note that this div is duplicated below. If making a change here, also make the change below. %>
-        <%= tag.div(class: classes, data: { index: nested_form.options[:child_index], nested_form_target: 'instance' }) do %>
+        <%= tag.div(class: row_classes, data: { index: nested_form.options[:child_index], nested_form_target: 'instance' }) do %>
```

Comment:
This allows us to reduce the amount of vertical space between repeatable fields such as keywords.